src/main/java/homeworks/juanpa/firstAssigment/composition/Tester.java

```diff
@@ -0,0 +1,41 @@
+package homeworks.juanpa.firstAssigment.composition;
+
+public class Tester {
+
+    private String role;
```

Comment:
if testers Class is being used as an abstract class for others to inherit it, you might wanna keep the access of these variables protected not private.